.github/workflows/test.yml

```
@@ -14,13 +14,13 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        go-version: [1.18.x, 1.19.x]
+        go-version: [1.21.x, 1.22.x]
```

For record, you can use stable and oldstable


```suggestion
        go-version: [oldstable, stable]
```

https://github.com/actions/setup-go?tab=readme-ov-file#using-stableoldstable-aliases

---

That’s smart!

But I wonder if there are any issues when 1.23 is released and it might unexpectedly break the build.

---

That's a risk. But a possible risk once compared to avoid the maintenance of this file

---

@ccoVeille Updated! What do you think about the other uses in this PR?

---

I think keeping a fixed version is a good thing for release.yml